GameServerLib/Content/NavGrid.cs

```diff
@@ -246,26 +243,26 @@ public void ToImage(string fileName)
             }
         }
 
-        public Vector<int> GetCellVector(short x, short y)
+        public Vector2 GetCellVector(short x, short y)
```

Comment:
Here you interchanged `int` by `float`. I don't think it is crucial, cause all this function need to get rework (it finds index in a rectangular grid using binary search, where you should use simple division and modulo). I need to check if it causes any problems.